test/extensions/INTEL/SPV_INTEL_fpga_latency_control/IntelFPGALatencyControl.ll

```diff
@@ -0,0 +1,75 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_fpga_latency_control -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: 2 Capability FPGALatencyControlINTEL
+; CHECK-SPIRV: 9 Extension "SPV_INTEL_fpga_latency_control"
+; CHECK-SPIRV: 4 Decorate [[ARGA:[0-9]+]] LatencyControlLabelINTEL 0
+; CHECK-SPIRV: 4 Decorate [[ARGB:[0-9]+]] LatencyControlLabelINTEL 1
+; CHECK-SPIRV: 6 Decorate [[ARGB:[0-9]+]] LatencyControlConstraintINTEL 0 1 5
+; CHECK-SPIRV: 4 Bitcast {{[0-9]+}} [[OUT1:[0-9]+]] [[ARGA]]
```

Comment:
nit, using [[#]] and [[#Variable:]] is a bit cleaner to parse literals in test's IR
```suggestion
; CHECK-SPIRV: Bitcast [[#]] [[#OUT1:]] [[#ARGA]]
```

Reply:
Applied this change wherever required. Good suggestion. Thanks.